### PR TITLE
Section 2

### PR DESCRIPTION
**Pull request by CheezItMan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,30 +54,49 @@ List all of the components
...
-The point of backbone is to provide a very intentional structure to our front-end code, so that we know where to find the code that is handling the specific functionality of our application.
+The point of Backbone is to provide a very intentional structure to our front-end code, so that we know where to find the code that is handling the specific functionality of our application.
 
-With Backbone being a less opinionated library/framework, it also has less rules and conventions on
+With Backbone being a less opinionated library/framework, it also has less rules and conventions on how to organize your application and where you implement specific functionality.  
 
+We'll take a look at a sample Rolodex solution.  
 
-### App.js
+Fork & Clone this [repository.](https://github.com/Hamled/rolodex/tree/Hamled/impl).
```
Comment (kariabancroft) [inline README.md]: I think many of them have already done this and I think they can probably get away without putting the code on their computer, right?

File: README.md [hunk 2]
```diff
@@ -86,11 +105,11 @@ To add more libraries, or to see which ones are already included, open the `pack
 
 If this process sounds familiar, it's because it's basically the same as having a gem file in rails and running *'bundle install'*.
 
-- **Webpack**
+- **Webpack**  Webpack lets us have a local webserver to run our app, translate our latest JavaScript syntax to something the browser can understand with **babel** and bundle our dependencies together.
 
-- **Underscore**
+- **Underscore** When you see an `_.` you're dealing with an underscore template.  Underscore provides a bunch or useful functions to use with Backbone, but we'll mainly focus on templating.
```
Comment (Hamled) [inline README.md]: I think we should make it clear that `_.` is *any* use of Underscore:

```markdown
- **Underscore** When you see an `_.` you're dealing with the Underscore library.  Underscore provides a bunch or useful functions to use with Backbone, but we'll mainly focus on templating by using the `template` function.
```

File: README.md [hunk 3]
```diff
@@ -86,11 +105,11 @@ To add more libraries, or to see which ones are already included, open the `pack
...
 If this process sounds familiar, it's because it's basically the same as having a gem file in rails and running *'bundle install'*.
 
-- **Webpack**
+- **Webpack**  Webpack lets us have a local webserver to run our app, translate our latest JavaScript syntax to something the browser can understand with **babel** and bundle our dependencies together.
 
-- **Underscore**
+- **Underscore** When you see an `_.` you're dealing with an underscore template.  Underscore provides a bunch or useful functions to use with Backbone, but we'll mainly focus on templating.  
 
-- **JQuery** Anytime you see that money sign, you're working with a Jquery object. That Jquery object is set by referring to a
+- **JQuery** Anytime you see that money sign, you're working with a Jquery object. That Jquery object is set by referring to a CSS selector for example $('body') to select the body element of the DOM.
```
Comment (Hamled) [inline README.md]: The jQuery library's name is actually stylized with a lowercase j and an uppercase Q.

File: README.md [hunk 4]
```diff
@@ -86,11 +105,11 @@ To add more libraries, or to see which ones are already included, open the `pack
 
 If this process sounds familiar, it's because it's basically the same as having a gem file in rails and running *'bundle install'*.
 
-- **Webpack**
+- **Webpack**  Webpack lets us have a local webserver to run our app, translate our latest JavaScript syntax to something the browser can understand with **babel** and bundle our dependencies together.
```
Comment (Hamled) [inline README.md]: "bundle our code and dependencies together into a single JavaScript file."

**Pull request merged**